src/RepositoryProxy.php

```diff
@@ -101,7 +101,15 @@ public function assertNotExists($criteria, string $message = ''): self
      */
     public function first(): ?Proxy
```

Comment:
```suggestion
    public function first(string $sort = 'id'): ?Proxy
```

The user could be using a different convention? (ie maybe UUIDs for the id want to use `createdAt` for the sort)